.github/workflows/pr_build.yml

```
@@ -0,0 +1,50 @@
+name: Pull Request CI Build
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build_en_docs:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    name: Build and upload english docs
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -U Sphinx==2.4.4
+        pip install --upgrade recommonmark
+        pip install sphinx_rtd_theme
+    - name: Build documentation
+      shell: bash
+      run: |
+        cd docs
+        make html
+        mkdir -p build-archives
+        cd build/html/
+        tar -czvf ../../build-archives/build-en.tar.gz *
+    - name: Upload documentation
+      uses: JulianGro/SFTP-Deploy-Action@v1.4
+      with:
+        username: 'gha'
+        server: '94.130.177.235'
+        ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
+        local_path: 'docs/build-archives/*'
+        remote_path: '/var/www/html/gha/PR${{ github.event.number }}/'
+        args: '-o ConnectTimeout=5'
+    - name: Deploy english documentation
+      shell: bash
+      run: |
+        printf "%s" "${{ secrets.SSH_PRIVATE_KEY }}" >private_key.pem
```

A space after the `>` would probably make it a bit more readable
```suggestion
        printf "%s" "${{ secrets.SSH_PRIVATE_KEY }}" > private_key.pem
```